lib/inputstreamhelper/__init__.py

```diff
@@ -93,7 +98,7 @@ def _temp_path(cls):
         return temp_path
 
     @classmethod
-    def _mnt_path(cls):
+    def _mk_mnt_path(cls):
```

Comment:
I don't like that this is renamed. The purpose of this function is to return the `_mnt_path()`, the fact that it creates it when missing is secondary to it returning the mount path.

Reply:
In the same light, I think we ought to call the function, rather than store it in a variable.

Reply:
My problem with that function is, that when e.g. `_cleanup()` is called on an x86 system, it runs `_unmount()`, which then calls `_mnt_path()`, creating a directory only to delete it again a second later. With my change this dir is only created when necessary.

Reply:
At first I thought about implementing it by using a variable `_stored_mnt_path = None`, which would be read by `_mnt_path()` if available, else it'd behave as usual. The thing is, that you can't specify a mount path to `udisksctl`, so it has to be read and stored somewhere.  When I realized the issue mentioned above, I changed it to how it is right now, but if you prefer I could implement it that way. I have a different idea how to fix this issue.

Reply:
No, you raise a valid point. Leave it as is.

Reply:
Have a look at my current implementation, that should fix this, too. I created another commit, so it's easy to revert. You can decide which version you prefer.